administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -16,6 +16,8 @@ COM_JOOMLAUPDATE_CONFIG_AUTOMATED_UPDATES_DISABLED_LABEL="<span class=\"fa-fw me
 COM_JOOMLAUPDATE_CONFIG_AUTOMATED_UPDATES_LABEL="Automated Updates"
 COM_JOOMLAUPDATE_CONFIG_AUTOUPDATE_DESC="Automatically update Joomla to the latest version when it is available."
 COM_JOOMLAUPDATE_CONFIG_AUTOUPDATE_LABEL="Automated Update"
+COM_JOOMLAUPDATE_CONFIG_AUTOUPDATE_UPDATE_EMAIL_DESCRIPTION="User groups that will receive emails about the automated updates. Per default all Super Users will receive the emails. Otherwise all users in the selected groups will receive the emails."
```

```suggestion
COM_JOOMLAUPDATE_CONFIG_AUTOUPDATE_UPDATE_EMAIL_DESCRIPTION="User groups that will receive emails about the automated updates. By default all Super Users will receive the emails. Otherwise all users in the selected groups will receive the emails."
```

simplify language level